test.js

```diff
@@ -1,36 +1,59 @@
 'use strict';
-var assert = require('assert');
+var assert = require('chai').assert;
```

Comment:
I received a little feedback in IRC, but not resolution: http://logs.nodejs.org/npm/latest#19:04:58.872

I'm for disabling the offending test, and filing an issue to follow back around with the issue later.